### ISSUE TITLE
[Feature Request]: Automatic/Toggle swap to normal YouTube

### ISSUE THREAD

**Issue by johnvo0918** [th-ch/youtube-music#1442]
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

I listen to DJ sets most of the time and a lot of them gets copyrighted (more like the sound is still there but youtube recognize that is a copyrighted song). and sometimes the songs that's available on normal youtube don't appear on youtube music (happens VERY rarely)

### Proposed Solution

Built in web browser/ player switcher. instead of going through music.youtube, the switch will go to regular youtube so that the DJ set with copyrighted song can still be played inside of your YTM app.

### Alternatives Considered

play from the website itself... but that defeats the purpose of having this application.
the only way i found to work around the missing songs that don't appear on youtube is by adding them into a playlist and praying that youtube music recognizes the playlist

**Comment by ArjixWasTaken** [comment]
I'm curious to see if we can rewrite the network requests to trick ytm into thinking they are normal songs.
I have no experience with the InnerTube API tho

**Comment by johnvo0918** [comment]
yeah i really enjoy using the application, it's also good to have a separate app for music on stream so that i can have it on its own track for twitch vods. really hoping to see this feature soon!

**Comment by johnvo0918** [comment]
Still waiting on this. I really don't want to use a whole browser just to listen to music, the app only takes 300mb of RAM to run while running a whole brave browser tab with the video player turned off via youtube audio only extension costs 3gb.
